src/content/en/agencies/_index.yaml

```diff
@@ -0,0 +1,95 @@
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+title: Find a qualified web dev agency to help you build optimized web apps
+description: >
+  These agencies have completed our advanced Web Apps Training for Agencies -
+  an intense course covering the Developing PWAs syllabus and hands-on codelabs.
+
+landing_page:
+  body_class: web-landing-page web-landing-page--agencies
+  custom_css_path: /web/agencies/css/web-agencies-styles.css
+  rows:
+
+  # INTRO
+  - classname: web-hero devsite-landing-row-50 devsite-landing-row-large-headings devsite-landing-row-no-image-background
+    items:
+    - heading: Web Apps Training for Agencies
+      description: >
+        An intense two-day course for web development agencies that covers the
+        <a href="https://developers.google.com/training/courses/progressive-web-apps">
+        Developing Progressive Web Applications</a> syllabus and several
+        hands-on codelabs. <br><br><b>Applications are reviewed based on:</b><br>
+        <ul><li>Perceived impact of agency’s client list</li>
+        <li>Agency size</li>
+        <li>Agency location (UK, India, and Indonesia for now)</li>
+        <li>Previous participation in training (two attendees per company
+        total)</li></ul>
+        <small>If you’re accepted, we’ll contact you with course dates and details.</small>
+      buttons:
+      - classname: gc-analytics-event button button-blue
+        label: Register your interest
+        path: https://docs.google.com/forms/d/e/1FAIpQLSfEr8PU68QeL5LFidmyHCZX3aEA5KhEN4kfDrGKw_kR1EaHvQ/viewform
+        attributes:
+        - name: data-category
+          value: Web Directory
+        - name: data-label
```

Comment:
for label, use something shorter, ideally hyphenated, eg: `agency-register` (and throughout)

Reply:
fyi, I've been labeling these values similar to how I've seen it in the rest of devsite (https://cs.corp.google.com/piper///depot/google3/googledata/devsite/content/en/programs/launchpad/_index.yaml?q=googledata/devsite/content/en/programs/launchpad/_index.yaml&sq=package:piper+file://depot/google3+-file:google3/experimental&l=42)

Typically they use button/link for value, and are a bit more descriptive when they come to the label and category.  I'm updating them regardless since it seems to be the way of webFu, but just wanted to add.

Reply:
Great thanks - since this goes into our analytics account, that's been our standard of doing things.